src/extension.c

```diff
@@ -1,12 +1,20 @@
 #include <postgres.h>
 #include <access/xact.h>
 #include <access/transam.h>
+#ifdef _MSC_VER
+#include "msvc/extension.h"
+#else
 #include <commands/extension.h>
+#endif // _MSC_VER
```

Comment:
Did you run this through pgindent. I believe we only support /**/ comments not //